From df0df0a0e2fef02e5cabee9edcee865d1d4580a2 Mon Sep 17 00:00:00 2001
From: Vlad Yasevich <vyasevic@redhat.com>
Date: Fri, 29 Aug 2014 19:01:22 -0400
Subject: [net] bridge: fix spelling of promiscuous

Message-id: <1409338885-30414-11-git-send-email-vyasevic@redhat.com>
Patchwork-id: 89904
O-Subject: [RHEL7 PATCH 10/13] bridge: fix spelling of promiscuous
Bugzilla: 896669
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Jiri Benc <jbenc@redhat.com>
RH-Acked-by: Hannes Frederic Sowa <hannes@redhat.com>

From: stephen hemminger <stephen@networkplumber.org>

Signed-off-by: Stephen Hemminger <stephen@networkplumber.org>
Signed-off-by: David S. Miller <davem@davemloft.net>
(cherry picked from commit 025559eec82cc170200d81cffcc7c2b231eb52b1)
Signed-off-by: Vladislav Yasevich <vyasevic@redhat.com>
Signed-off-by: Jiri Benc <jbenc@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/net/bridge/br_if.c b/net/bridge/br_if.c
index 71ed0d5..dddb8bf 100644
--- a/net/bridge/br_if.c
+++ b/net/bridge/br_if.c
@@ -178,7 +178,7 @@ static void nbp_update_port_count(struct net_bridge *br)
 
 static void nbp_delete_promisc(struct net_bridge_port *p)
 {
- /* If port is currently promiscous, unset promiscuity.
+ /* If port is currently promiscuous, unset promiscuity.
   * Otherwise, it is a static port so remove all addresses
   * from it.
   */
-- 
1.7.1